packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -422,30 +434,32 @@ export const ChatScreen = () => {
               hasCurrentlyPlayingTrack ? { bottom: PLAY_BAR_HEIGHT } : null
             ]}
           >
+            {chat?.messagesStatus === Status.SUCCESS &&
+            chatMessages?.length === 0 ? (
+              <EmptyChatMessages />
+            ) : null}
             {!isLoading ? (
```

Comment:
Minor nit: Negated conditionals are a little harder to read. In the future, could arrange the ternary the other way, so that the loading component comes first and you can do `isLoading ? <LoadingComponent /> : <ContentComponent />`